plugins/chat/assets/javascripts/discourse/services/chat.js

```diff
@@ -294,68 +320,6 @@ export default class Chat extends Service {
     }
   }
 
-  getIdealFirstChannelId() {
```

Comment:
ALl of this is no necessary anymore has we use the user custom field to get last id